runcoms/zshenv

```diff
@@ -6,6 +6,6 @@
 #
 
 # Ensure that a non-login, non-interactive shell has a defined environment.
-if [[ "$SHLVL" -eq 1 && ! -o LOGIN && -s "${ZDOTDIR:-$HOME}/.zprofile" ]]; then
+if [[ "$SHLVL" -eq 1 && ! -o LOGIN && -s "${ZDOTDIR:-$HOME}/.zprofile" || ! "${TMPDIR}" ]]; then
```

Comment:
No issue with changing it. Just mentioning it as something to keep in mind. https://github.com/sorin-ionescu/prezto/issues/1180